samples/grids/data-grid/load-save-layout/src/index.tsx

```diff
@@ -0,0 +1,378 @@
+
+        const peopleCount = new IgrColumnSummaryDescription();
+        peopleCount.field = "Photo";
+        peopleCount.operand = SummaryOperand.Count;
+        this.grid.summaryDescriptions.add(peopleCount);
+
+        const sales = new IgrColumnSummaryDescription();
+        sales.field = "Sales";
+        sales.operand = SummaryOperand.Max;
+        sales.formatOverride = new Intl.NumberFormat('en-EN', { style: 'currency', currency: 'USD', minimumFractionDigits: 0, maximumFractionDigits: 0 });
+        this.grid.summaryDescriptions.add(sales);
+
+        const salary = new IgrColumnSummaryDescription();
+        salary.field = "Salary";
+        salary.operand = SummaryOperand.Average;
+        salary.formatOverride = new Intl.NumberFormat('en-EN', { style: 'currency', currency: 'USD', minimumFractionDigits: 0, maximumFractionDigits: 0 });
+        this.grid.summaryDescriptions.add(salary);
+    }
+
+    public getProductivityChart(props: IIgrCellTemplateProps) {
+        const info = props.dataContext as IgrTemplateCellInfo;
+        return (
+            <div className="container">
+               <IgrSparkline
+                   height="30px" width="100%"
+                   displayType="Column"
+                   dataSource={info.rowItem.Productivity}
+                   valueMemberPath="Value"
+                   labelMemberPath="Week"
+                   lineThickness={2}
+                   brush="rgb(21, 190, 6)"
+                   negativeBrush="rgb(211, 17, 3)" />
+            </div>
+        );
+    }
+
+    public onAddressCellUpdating(s: IgrTemplateColumn, e: IgrTemplateCellUpdatingEventArgs) {
+        const content = e.content as HTMLDivElement;
+        let span1: HTMLSpanElement | null = null;
+        let span2: HTMLSpanElement | null = null;
+        const info = e.cellInfo as IgrTemplateCellInfo;
+        const item = info.rowItem;
+
+        if (content.childElementCount === 0) {
+
+            span1 = document.createElement("span");
+            span2 = document.createElement("span");
+
+            content.style.fontFamily = "Verdana";
+            content.style.fontSize = "13px";
+            content.style.verticalAlign = "center";
+            content.style.lineHeight = "normal";
+            content.style.display = "flex";
+            content.style.flexDirection = "column";
+            content.style.justifyContent = "center";
+            content.style.height = "100%";
+            content.style.width = "100%";
+            content.appendChild(span1);
+            content.appendChild(span2);
+        }
+        else {
+            span1 = content.children[0] as HTMLSpanElement;
+            span2 = content.children[1] as HTMLSpanElement;
+        }
+
+        if (span1 && span2) {
+            span1.textContent = item.Street;
+            span2.textContent = item.City + ", " + item.Country;
+        }
+    }
+
+    public onSalesCellUpdating(s: IgrTemplateColumn, e: IgrTemplateCellUpdatingEventArgs) {
```

Comment:
remove this function to make this sample less complex

Reply:
resolved

https://github.com/IgniteUI/igniteui-react-examples/pull/101/commits/af54a47b9c02875025df7f83cf81dc8e6813eb85